src/resources/postcss/aggregator-page.pcss

```diff
@@ -252,7 +252,57 @@
 	}
 
 	.tribe-notice-tribe-missing-aggregator-license {
-		background: #f7fcfe;
+		background: url(../images/aggregator/ea-upsell-bkg.svg) transparent no-repeat;
+		background-size: cover;
+		border: none;
+		padding: 60px 0;
+		text-align: center;
+
+		.upsell-banner {
+			margin-bottom: 30px;
+			margin-left: auto;
+			margin-right: auto;
+		}
+
+		h3 {
+			color: white;
```

Comment:
`#fff` is ever so slightly more efficient in that it is 1 character shorter. Probably a negligible performance improvement, but that is immediately where my mind goes. If you absolutely feel that `white` is preferred, then I can acquiesce.

My brain reads `#fff` as easily as `white`, so my preference is for the former.